<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvenementsOrbitaux</class>
 <widget class="QFrame" name="EvenementsOrbitaux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="listeEvenements">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>43</y>
     <width>210</width>
     <height>164</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxEvenements">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Évènements</string>
   </property>
   <widget class="QCheckBox" name="passageNoeuds">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Passages aux noeuds</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="passageQuadrangles">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Passages à PSO = 90° et 270°</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="passageApogee">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Passages apogée/périgée</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="passageOmbre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Passages ombre/pénombre/lumière</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="transitionJourNuit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Passages terminateur</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>338</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_dateEvt">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_dateFinaleEvt">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date finale :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateFinaleEvt">
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_dateInitialeEvt">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date initiale :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateInitialeEvt">
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="effacerHeuresEvt">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Effacer heures</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calculsEvt">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>185</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Calculs</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="parametrageDefautEvt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Paramétrage par défaut</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filtreSatellites">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>20</y>
     <width>210</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Filtre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
